internal/lsp/configuration.go

```diff
@@ -22,3 +23,31 @@ func (s *Server) Config(section string) interface{} {
 	}
 	return v
 }
+
+var regFormatCapability = protocol.RegistrationParams{
+	Registrations: []protocol.Registration{
+		{ID: "FormCap",
```

Comment:
"FormCap" seems arbitrary and ambiguous, how about something like "ttcn3.formatting" instead?